src/v3/test/integration/__snapshots__/authenticator-expired-password-with-enrollment-authenticator.test.tsx.snap

```diff
@@ -105,15 +106,15 @@ exports[`authenticator-expired-password-with-enrollment-authenticator should ren
                           class="ods-4o5zYQ ods-n3PBAA ods-35YmB2 ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs passwordRequirementIcon info"
                         >
                           <svg
-                            aria-labelledby="generated"
+                            aria-labelledby="dq3b6q"
                             class="ods-2icygl"
                             fill="none"
                             role="img"
                             viewBox="0 0 16 16"
                             xmlns="http://www.w3.org/2000/svg"
                           >
                             <title
-                              id="generated"
+                              id="dq3b6q"
```

Comment:
These values are getting auto generated, any way to prevent this?

Reply:
ohh, seems like we are still with odyssey component here, please use that mask function then, I thought all components have been replaced